packages/omg-js-util/src/transaction.js

```diff
@@ -80,21 +113,53 @@ const transaction = {
     })
   },
 
-  createTransactionBody: function (fromAddress, fromUtxos, toAddress, toAmount) {
+  /**
+  * Creates a transaction object. It will select from the given utxos to cover the amount
+  * of the transaction, sending any remainder back as change.
+  *
+  *@param {string} fromAddress the address of the sender
+  *@param {string} fromUtxos the utxos to use as transaction inputs
+  *@param {string} toAddress the address of the receiver
+  *@param {string} toAmount the amount to send
+  *@param {string} currency the currency to send
+  *@returns the transaction object
+  *@throws InvalidArgumentError if any of the args are invalid
+  *@throws Error if the given utxos cannot cover the amount
+  *
+  */
+  createTransactionBody: function (fromAddress, fromUtxos, toAddress, toAmount, currency) {
     validateInputs(fromUtxos)
-    const inputArr = fromUtxos.map(utxo => utxo)
+    let inputArr = fromUtxos.filter(utxo => utxo.currency === currency)
+
+    // We can use a maximum of 4 inputs, so just take the largest 4 inputs and try to cover the amount with them
+    // TODO Be more clever about this...
```

Comment:
hmmm, we might need to be explicit about different strategy to `createTransactionBody` in the future. How else can we be more clever about this ?

Reply:
I guess I'm considering `createTransactionBody` as a kind of poor man's utxo selection function for now. When we have real utxo selection/management, I don't think people will use it.